doc/proposals/ansible-operator-webhooks.md

```diff
@@ -0,0 +1,81 @@
+# Ansible Operator Webhooks
+
+> Status: Draft
+    are necessary to support defaulting, though the addition of the default field to the OpenAPI spec may make this one 
+## Background
+- [Conversion webhooks](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion)
+    are necessary to support CRD versioning
+- [Mutating webhooks](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)
+    are necessary to support defaulting, though the addition of the default field to the OpenAPI spec may make this one 
+    less important in the long-term.
+- [Validating webhooks](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) 
+    are necessary for higher level validation of incoming resources, for example SAR checks. There is an open community 
+    issue explicitly requesting this feature: https://github.com/operator-framework/operator-sdk/issues/1658 .
+
+Currently Ansible-based operators have no webhook support at all, a user would have to write their own webhook server 
+and deploy it alongside the operator to get that functionality.
+
+## Goals
+- Allow the creation and use of conversion/validating/mutating webhooks implemented with Ansible
```

Comment:
IMHO the goals should be some which will describe the stories. E.g A operator user ansible developer should be able to ... in order to make it clear. WDYT?